Sources/RswiftCore/Generators/StringsStructGenerator.swift

```diff
@@ -285,6 +308,43 @@ struct StringsStructGenerator: ExternalOnlyStructGenerator {
       defaultValue: "nil"
     )
     params.append(prefereredLanguages)
+    
+    let body: String
```

Comment:
This does not seem very DRY. Is there a way for us to isolate just the part that changes? (ie: the nullability check?) Maybe consider composing body in two steps where in the first step you define
```swift
var body: String
switch bundle {
    case .hostingBundle:      
        body = """
        let bundle = \(bundle)
        """
    case .customBundle:
      body = """
        guard let bundle = \(bundle) else {
          return "\(values.key.escapedStringLiteral)"
        }
       """
    }

body += """
 
       guard let preferredLanguages = preferredLanguages else {
          let format = \(values.swiftCode(bundle: "bundle"))
          return String(format: format, locale: applicationLocale, \(args))
        }
        guard let (locale, foundBundle) = localeBundle(parentBundle: bundle, tableName: "\(values.tableName)", preferredLanguages: preferredLanguages) else {
          return "\(values.key.escapedStringLiteral)" 
        }
        let format = \(values.swiftCode(bundle: "foundBundle"))
        return String(format: format, locale: locale, \(args))
        """
    }
```

You could even move the switch into an extension of BundleExpression to avoid having to type it twice then!